File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/colorselector/AlphaSelectionListener.java
package net.kdt.pojavlaunch.colorselector;

/**
 * アルファ（透明度）値が選択されたときのリスナーインターフェース。
 */
public interface AlphaSelectionListener {
    /**
     * アルファ値が選択されたときに呼び出されます。
     * @param alpha 選択されたアルファ値（0〜255）
     */
    void onAlphaSelected(int alpha);
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/ui/view/colorselector/AlphaSelectionListener.kt
package com.arata.yukarilauncher.ui.view.colorselector

/** アルファ値（透明度）が選択された際のコールバックインターフェース。 */
interface AlphaSelectionListener {
    /** アルファ値が選択されたときに呼び出される。 @param alpha 選択されたアルファ値（0～255） */
    fun onAlphaSelected(alpha: Int)
}